### PR TITLE
Survey report

### PR DESCRIPTION
**Pull request by ljwoodley**
Basically copied the functions from https://github.com/ctsit/warrior_reports/blob/master/R/create_column_labels.R .

### CODE REVIEW

**Comment by pbchase** [comment]
@ljwoodley, I want to assign this one to @ChemiKyle

File: functions.R [hunk 1]
```diff
@@ -0,0 +1,99 @@
+library(tidyverse)
+library(janitor)
+
+data_dictionary <- list.files("data", "FirstResponderCOVID19TestingPr")
+data_dictionary <- read.csv(paste0("data/", data_dictionary)) %>% 
+  clean_names()
```
Comment (ChemiKyle) [inline functions.R]: This file is not included in your PR. @pbchase suggests fetching this with the API if possible.

**Comment by ChemiKyle** [comment]
The report to compile is hardcoded as the old `appointment_report.Rmd` in this PR:  
https://github.com/ljwoodley/covid19_reporting/blob/68cf8343c0668edde18c010fe7dcc1dee62ce9f4/render_report.R#L7

Changing it manually to `survey_report.Rmd` strangely emails me the output of `appointment_report.Rmd`.

Which report to compile should be set as a variable, you could pass it via the `env` file or as an arg in the `docker run`.

**Comment by ljwoodley** [comment]
@ChemiKyle that's because the file name will be the same.https://github.com/ljwoodley/covid19_reporting/blob/68cf8343c0668edde18c010fe7dcc1dee62ce9f4/render_report.R#L6
Also survey_report is a one time report for now I think. That's why I didn't add it to render_report.R

**Comment by ljwoodley** [comment]
I will look into reading the data dictionary with the API later. For now we can download it manually as this needs to go out soon.

File: functions.R (hunk 1 above)
Comment (ljwoodley) [reply functions.R]: I will look into reading the data dictionary with the API later. For now we can download it manually as this needs to go out soon.

**Comment by ljwoodley** [comment]
@ChemiKyle Philip caught me up on the issues. You'll have to download the data dictionary from REDCap and place it in the data folder. I will do the automation and figure out how to read the data dictionary via the api (if possible) once I'm freed up.  If you'd like to compare the counts in the report to the raw dataset, for a given column you can do `table(records$col_name)`. If you got any more questions let me know.

**Comment by ljwoodley** [comment]
Here are the fields @pbchase wanted copied from baseline.
Screen Shot 2020-04-29 at 5 18 01 PM

Additionally, I added name and dob.

**Comment by ljwoodley** [comment]
Hold on this. There's a function to read the data dictionary. Adding it now

**Comment by ljwoodley** [comment]
This is ready for review.

**Pull request merged**